modules/oe_theme_helper/tests/src/Functional/RetinaScaleEffectTest.php

```diff
@@ -30,80 +44,91 @@ class RetinaScaleEffectTest extends ToolkitTestBase {
   /**
    * {@inheritdoc}
    */
-  protected function setUp() {
+  protected $defaultTheme = 'stark';
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function setUp(): void {
     parent::setUp();
     $this->manager = $this->container->get('plugin.manager.image.effect');
+    $this->image = $this->getImage();
+    $this->imageTestReset();
   }
 
   /**
    * Test the retina scale effect using a big enough image.
    */
   public function testRetinaScaleEffect(): void {
-    $this->assertImageEffect('retina_image_scale', [
+    $this->assertImageEffect(['scale'], 'retina_image_scale', [
       // Set the desired width to be smaller than the image width.
       'width' => 10,
       'height' => 10,
     ]);
     $this->assertToolkitOperationsCalled(['scale']);
 
     $calls = $this->imageTestGetAllCalls();
-    $this->assertEqual($calls['scale'][0][0], 10, 'Width was passed correctly');
-    $this->assertEqual($calls['scale'][0][1], 10, 'Height was based off aspect ratio and passed correctly');
+    $this->assertEquals(10, $calls['scale'][0][0], 'Width was passed correctly');
+    $this->assertEquals(10, $calls['scale'][0][1], 'Height was based off aspect ratio and passed correctly');
```

Comment:
It's correct to swap the params, as in PHPUnit the expectation comes first. However, the messages are not OK. In PHPUnit we tell the failure not the success. But I'm OK also to remove the message.